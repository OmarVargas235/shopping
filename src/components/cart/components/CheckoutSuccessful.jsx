import React from 'react';
import styled from 'styled-components';
import { Link } from 'react-router-dom';

const Container = styled.div`
	.checkout {
		color: #4bbf73;
	}

	.btn--buy {
		border: 2px solid #4bbf73;
		font-size: .9rem;
		background-color: transparent;
		color: #4bbf73;

		&:hover {
			background-color: #4bbf73;
    		color: #fff;
		}

		&:focus {
			box-shadow: 0 0 0 0.2rem #37B161;
		}
	}
`;

const CheckoutSuccessful = () => (
	<Container className="text-center">
		<p className="checkout">Checkout successfull</p>

		<Link className="btn btn--buy py-3" to='/home'>BUY MORE</Link>
	</Container>
)

export default CheckoutSuccessful;