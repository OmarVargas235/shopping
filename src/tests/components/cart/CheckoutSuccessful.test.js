import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import toJson from 'enzyme-to-json';
import CheckoutSuccessful from '../../../components/cart/components/CheckoutSuccessful';

test('Comprobando el marcado html en <CheckoutSuccessful />', () => {	
	const wrapper = shallow( <CheckoutSuccessful /> );

    expect(toJson(wrapper)).toMatchSnapshot();
});