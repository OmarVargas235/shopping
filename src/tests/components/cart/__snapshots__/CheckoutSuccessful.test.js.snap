// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comprobando el marcado html en <CheckoutSuccessful /> 1`] = `
<styled.div
  className="text-center"
>
  <p
    className="checkout"
  >
    Checkout successfull
  </p>
  <Link
    className="btn btn--buy py-3"
    to="/home"
  >
    BUY MORE
  </Link>
</styled.div>
`;
